playbooks/ci-full-centos/templates/heat_stack.yml

```diff
@@ -8,19 +8,19 @@ parameters:
   image:
     type: string
     description: Name of image to use for servers
-    default: {{ ansible_env.IMAGE_ID | default('ubuntu-14.04') }}
+    default: centos7
```

Comment:
can change this to reference by image-id 11760201-107c-4387-a54f-ec738533d71f
i.e
default: 11760201-107c-4387-a54f-ec738533d71f